src/test/java/org/folio/edge/controller/InnReachProxyControllerTest.java

```diff
@@ -98,4 +101,22 @@ void shouldUpdateRequestHeaders() {
     );
   }
 
+  @Test
+  void shouldHandleCommonErrors() {
+    var httpHeaders = createInnReachHttpHeaders();
+    var requestEntity = new HttpEntity<>(httpHeaders);
+
+    wireMock.stubFor(post(PATRON_VERIFY_URL_PATTERN)
+      .willReturn(aResponse().withBody("{}")
```

Comment:
set some INN-reach error as body and assert if it serializes properly

Reply:
Fixed